src/main/java/ro/societateahermes/backendservice/controller/ActivityController.java

```diff
@@ -0,0 +1,39 @@
+package ro.societateahermes.backendservice.controller;
+
+import org.springframework.web.bind.annotation.*;
+import ro.societateahermes.backendservice.entities.Activity;
+import ro.societateahermes.backendservice.entities.DTO.ActivityDTO;
+import ro.societateahermes.backendservice.service.ActivityServiceInterface;
+import ro.societateahermes.backendservice.service.serviceImplementation.ActivityServiceImplementation;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/activity")
+public class ActivityController implements ActivityControllerInterface{
+    private ActivityServiceInterface activityService;
+
+    public ActivityController(ActivityServiceImplementation activityService){
+        this.activityService = activityService;
+    }
+
+    @PostMapping
+    public void save(@RequestBody Activity activity) {
```

Comment:
E bine sa folosesti un dto ca sa primesti si sa trimiti date din si in frontend modifica peste tot unde folosesti Activity astfel incat sa folosesti un ActivityDto care sa contina strict datele de care ai nevoie